Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM ruby:2.6.5 as builder
 
 # Install Java
-RUN curl -L -o /tmp/jdk.tar.gz https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/download/jdk8u232-b09_openj9-0.17.0/OpenJDK8U-jdk_x64_linux_openj9_8u232b09_openj9-0.17.0.tar.gz \
+RUN curl -L -o /tmp/jdk.tar.gz https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/download/jdk8u242-b08_openj9-0.18.1/OpenJDK8U-jdk_x64_linux_openj9_8u242b08_openj9-0.18.1.tar.gz \
```

Comment:
looks like the current latest openJDK (8 hrs ago) is https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/tag/jdk8u252-b09.1_openj9-0.20.0 so lets try that one out